Dockerfile

```diff
@@ -1,33 +1,38 @@
 FROM registry.access.redhat.com/ubi8/ubi-minimal:latest
 WORKDIR /bin
 
-RUN microdnf install --nodocs tar gcc gzip git bind-utils sudo \
+RUN microdnf install --nodocs tar gcc gzip git bind-utils findutils sudo \
     && curl -Lo ./oc.tar.gz https://mirror.openshift.com/pub/openshift-v4/clients/ocp/stable/openshift-client-linux.tar.gz \
     && tar -xf oc.tar.gz \
     && rm -f oc.tar.gz \
-    && curl -Lo ./golang.tar.gz https://go.dev/dl/go1.16.15.linux-amd64.tar.gz \
+    && curl -Lo ./golang.tar.gz https://go.dev/dl/go1.19.8.linux-amd64.tar.gz \
```

Comment:
Any reason not to use the latest stable version 1.20.3?

Reply:
fixed.